setup.py

```diff
@@ -230,20 +235,8 @@ def _get_package_path(package_name):
     return None
 
 
-BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"
```

Comment:
Can we unify this one and https://github.com/pytorch/pytorch/pull/126328 ? Since these lines seems to be added by #126328 ?

Reply:
These two PRs are wholly unrelated. The only reason they're together in a stack is to not have a merge conflict. I can just remove this PR and CI should still pass if you want